stylesheets/about.css

```diff
@@ -1 +1,163 @@
-/* This is an empty file */
+/* This file contains all the styling for the landing page for the about page of The VSD project */
+
+/* The mobile-first approach was used in styling. */
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused code so we can have a clean code-base. It is a good practice to have a clean code-base.